docs/data_sources/sensu_cluster_role.md

```diff
@@ -0,0 +1,31 @@
+# sensu_cluster_role
+
+Get information about a Sensu Cluser Role.
+
+For full documentation on Sensu Cluster Roles, see [here](https://docs.sensu.io/sensu-go/5.20/reference/rbac/#roles-and-cluster-roles).
+
+## Basic Example
+
+```hcl
+data "sensu_cluster_role" "cluster_role_1" {
+  name = "my_role"
+}
+```
+
+## Argument Reference
+
+* `name` - *Required* - The name of the Sensu cluster role.
+
+## Attribute Reference
+
+* `rule` - *Required* - See the [Sensu rbac reference](https://docs.sensu.io/sensu-go/latest/reference/rbac/#rule-attributes).
```

Comment:
Attributes (or outputs) aren't required - we can remove the word `*Required*` here.